frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -885,75 +889,83 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
-                              </GridItem>
-                              {ns.name === serverConfig.istioNamespace &&
-                                <GridItem md={9}>
-                                  <Grid>
-                                    {this.state.canaryUpgradeStatus && this.hasCanaryUpgradeConfigured() &&
-                                      <GridItem md={4}>
-                                        <CanaryUpgradeProgress canaryUpgradeStatus={this.state.canaryUpgradeStatus} />
-                                      </GridItem>
-                                    }
-                                    <GridItem md={(this.hasCanaryUpgradeConfigured()) ? 8 : 12}>
-                                      {this.renderCharts(ns)}
+                                      )}
+                                      {this.renderIstioConfigStatus(ns)}
+                                    </div>
+                                    {ns.status && <NamespaceStatuses key={ns.name} name={ns.name} status={ns.status}
+                                                                     type={this.state.type}/>}
+                                    {this.state.displayMode === OverviewDisplayMode.EXPAND &&
+                                      <ControlPlaneNamespaceStatus outboundTrafficPolicy={this.state.outboundPolicyMode}
+                                                                   namespace={ns}></ControlPlaneNamespaceStatus>}
+                                    {this.state.displayMode === OverviewDisplayMode.EXPAND && <TLSInfo
+                                      certificatesInformationIndicators={serverConfig.kialiFeatureFlags.certificatesInformationIndicators.enabled}
+                                      version={this.props.minTLS}></TLSInfo>}
+                                  </GridItem>
+                                  {ns.name === serverConfig.istioNamespace &&
+                                    <GridItem md={9}>
+                                      <Grid>
+                                        {this.state.canaryUpgradeStatus && this.hasCanaryUpgradeConfigured() &&
+                                          <GridItem md={4}>
+                                            <CanaryUpgradeProgress
+                                              canaryUpgradeStatus={this.state.canaryUpgradeStatus}/>
+                                          </GridItem>
+                                        }
+                                        {this.props.istioAPIEnabled === true &&
+                                          <GridItem md={(this.hasCanaryUpgradeConfigured()) ? 8 : 12}>
+                                            {this.renderCharts(ns)}
+                                          </GridItem>}
```

Comment:
Minor: You don't need to resolve this as part of this PR but there's now a bunch of whitespace on the controlplane card without the metrics:

![Screenshot from 2022-12-19 21-02-48](https://user-images.githubusercontent.com/6226732/208565288-78b319a0-4fbd-43e9-b693-eb4cd530c052.png)

I'm not sure what to fill that space with.